syntaxes/chapel.tmLanguage.json

```diff
@@ -29,7 +29,7 @@
     "keywords": {
       "patterns": [
         {
-          "match": "\\b(align|as|atomic|begin|break|by|catch|class|cobegin|coforall|deinit|continue|defer|delete|dmapped|do|else|enum|except|export|extern|for|forall|foreach|forwarding|if|import|in|index|init|init=|inline|inout|interface|implements|iter|label|lambda|let|lifetime|local|manage|module|new|noinit|on|only|operator|otherwise|out|override|postinit|__primitive|pragma|private|proc|prototype|public|record|reduce|require|return|scan|select|serial|shared|then|these|throw|throws|try|union|use|when|where|while|with|yield|zip)\\b",
+          "match": "\\b(align|as|atomic|begin|break|by|catch|class|cobegin|coforall|deinit|continue|defer|delete|dmapped|do|else|enum|except|export|extern|for|forall|foreach|forwarding|if|import|in|include|index|init|init=|inline|inout|interface|implements|iter|label|lambda|let|lifetime|local|manage|module|new|noinit|on|only|operator|otherwise|out|override|postinit|__primitive|pragma|private|proc|prototype|public|record|reduce|require|return|scan|select|serial|shared|then|these|throw|throws|try|union|use|when|where|while|with|yield|zip)\\b",
```

Comment:
This is not introduced by your PR, but since we are here, why is `shared` classified as a control keyword whereas `owned` and `borrowed` are in the storage modifier category in this file?

Reply:
That's a great question...I don't know 😄.

That's gonna bug me, so I'll fix it now.

Reply:
Ok I do actually know why. There was also a duplicated `sync`, because `sync` is "ambiguous", it can either be a type or a keyword on a block. And I have spent more time than I care to admit trying to get them highlighted differently using only regex. But this typo with shared seems like me making a mistake while messing with sync....